worker/docker_client.py

```diff
@@ -224,19 +224,10 @@ def start_container(self, bundle_path, uuid, command, docker_image,
         # Set up the command.
         docker_bundle_path = '/' + uuid
         docker_commands = [
-            'BASHRC=$(pwd)/.bashrc',
-            # We pass several commands for bash to execute as a single
-            # argument (i.e. all commands appear in quotes with no spaces
-            # outside the quotes). The first commands appear in double quotes
-            # since we want environment variables to be expanded. The last
-            # appears in single quotes since we do not. The expansion there,
-            # if any, should happen when bash executes it. Note, since the
-            # user's command can have single quotes we need to escape them.
-            'bash -c '
-            + '"[ -e $BASHRC ] && . $BASHRC; "'
-            + '"cd %s; "' % docker_bundle_path
-            + '"export HOME=%s; "' % docker_bundle_path
-            + '\'(%s) >stdout 2>stderr\'' % command.replace('\'', '\'"\'"\''),
+            # TODO: Remove when codalab/torch uses ENV directives
```

Comment:
Oh, we're so close to having this be clean!

But I think this is not going to work for the codalab/ubuntu:1.9 image either, which relies on sourcing .bashrc before cd'ing into docker_bundle_path...

Probably need to have to source .bashrc that exists in several possible directories...
Can you try this out on several docker images to make sure that they all work?

This should be part of our test - even if it is not a formal process...just have a list of docker images that we commit ourselves to checking when we make changes like this.